include/libhat/scanner.hpp

```diff
@@ -163,7 +166,7 @@ namespace hat::detail {
 
         template<scan_mode mode = scan_mode::Auto>
         static constexpr scan_context create(signature_view signature, scan_alignment alignment, scan_hint hints);
-    private:
+        
```

Comment:
Reconsider refactoring the `hat::scanner` constructor instead of just making this public